src/nrnpython/nrnpy_hoc.cpp

```diff
@@ -2900,53 +2900,31 @@ static PyObject* hocpickle_reduce(PyObject* self, PyObject* args) {
     PyHocObject* pho = (PyHocObject*) self;
     if (!is_obj_type(pho->ho_, "Vector")) {
         PyErr_SetString(PyExc_TypeError, "HocObject: Only Vector instance can be pickled");
-        return NULL;
+        return nullptr;
     }
     Vect* vec = (Vect*) pho->ho_->u.this_pointer;
 
     // neuron module has a _pkl method that returns h.Vector(0)
-    auto mod = nb::steal(PyImport_ImportModule("neuron"));
-    if (!mod) {
```

Comment:
This check is being skipped, since it's likely uncovered by unit-tests, would it be better to keep the check?